src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -125,17 +120,21 @@ Standard distributed tracing for APM agents ([above](#quick-start-apm)) captures
 To learn more about this feature, see [Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing).
 </Callout>
 
-Before beginning, first ensure you meet [the requirements](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing#requirements).
-
-#### Step 1. Complete the new agent installation [#install-agents-first]
+Before you start, first ensure you meet [the requirements](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing#requirements).
 
-The Infinite Tracing setup builds on the instrumentation step from the [new agent installation](#quick-start-apm) for standard distributed tracing. After you finish installing the agents, continue with the trace observer setup.
+<Steps>
+  <Step>
+### Complete the setup for standard distributed tracing [#install-agents-first]
 
-#### Step 2. Set up the trace observer [#provision-trace-observer]
+The Infinite Tracing setup builds on standard distributed tracing. So, make sure you've completed the [steps above](#quick-start-apm), and then continue with the trace observer setup.
+  </Step>
+  <Step>
+### Set up the trace observer [#provision-trace-observer]
 
 The trace observer is a New Relic AWS-based service that collects and analyzes all your traces. Follow the instructions in <a href="/docs/understand-dependencies/distributed-tracing/infinite-tracing/set-trace-observer" target="_blank">Set up trace observer</a>. When you're done, return here with your trace observer information and continue with the next step to configure the agent.
```

Comment:
Here there is another HTML structure for URL.

Reply:
@nbaenam Good catch! I fixed this!